docs/src/docs/cli/installation.md

```diff
@@ -53,25 +53,25 @@ That's it!
 
 Make sure you get the correct download for your hardware.
 
-1. <a href="https://github.com/jozu-ai/kitops/releases/latest/download/kitops-windows-x86_64.zip"
+1. Windows: <a href="https://github.com/jozu-ai/kitops/releases/latest/download/kitops-windows-arm64.zip"
```

Comment:
We probably want x86 to be first for Windows. There are not many arm machines that run windows normally.

Reply:
fixed